### PR TITLE
Jupyter interactive mode

### PR DESCRIPTION
**Pull request by Jagadeesh-ajjada**
Project code is added to the Jupyter Notenook
Note: Currently, the code runs when we start only the Jupyter server. But, if we run both the Streamlit and Jupyter servers simultaneously, an error occurs in Jupyter, indicating that Milvus is already in use by the other server(Streamlit). Am pushing this since we have demo on Tuesday, However some changes are still needed for it to work on both servers.

- After build, run this command `docker run -d -p 6001:6001 team1_app` inster of this `docker run -d -p 5001:5001 -p 6001:6001 team1_app`

### CODE REVIEW

File: documentation.ipynb [hunk 1]
```diff
@@ -4,8 +4,335 @@
...
+    "\n",
+    "def load_documents_from_web():\n",
+    "    \"\"\"\n",
+    "    Load the documents from the web and store the page contents\n",
+    "\n",
+    "    Returns:\n",
+    "        list: The documents loaded from the web\n",
+    "    \"\"\"\n",
+    "    loader = RecursiveUrlLoader(\n",
+    "        url=CORPUS_SOURCE,\n",
```
Comment (smrchanda877) [inline documentation.ipynb:151]: please define `CORPUS_SOURCE` before this (probably in the first or second block where we are importing everything)

**Review by smrchanda877 — CHANGES_REQUESTED** [review]

**Comment by smrchanda877** [comment]
@Jagadeesh-ajjada if we are getting the error that `milvus` is already used by `streamlit`, do you think we can use another milvus vector store? Like how we are creating and using for streamlit, we'll just give another `MILVUS_URI` in jupyter.
If it takes too much time in demo to download embeddings and embed the data, we can just do that now, and download the database in the same way we did for `streamlit` milvus.

**Comment by Jagadeesh-ajjada** [comment]
Yeah @smrchanda877 CORPUS_SOURCE added to the code.

**Comment by Jagadeesh-ajjada** [comment]
@smrchanda877 @itsnotmik @ACraig7 Can you review this, as of now separated database file added to Jupyter make sure no db conflicts, i'll research if we can do it with single db config.

**Comment by ACraig7** [comment]
I am currently running it, I did the single command with both ports (5001 and 6001) and it has to go through the embedding process, which is not ideal. I also ran it with two commands, port 6001 first and port 5001 second, and it still has to go through the embedding process. I do not think we should merge this in its current form cause it is causing the embedding process to happen every time which makes testing future commits harder.

**Comment by smrchanda877** [comment]
Oh, is it loading embeddings? when I tested on my end, as `jupyter_milvus_vecor.db` is already present in the branch, it's not taking much time. Can you please verify? @ACraig7

**Comment by itsnotmik** [comment]
I don't believe its loading embeddings. I was able to run the commands all and get a response. I think the .db file is working correctly

**Comment by itsnotmik** [comment]
Maybe @ACraig7 pulled the branch right before the new commit went through that contained the .db files

**Comment by itsnotmik** [comment]
There is an error in this cell though. It should be fully deleted since the code present here is actually present in the previous cell

image

The cell begins with "dings loaded", which is present in the previous cell when it loads the embeddings

**Comment by itsnotmik** [comment]
I have fixed on newest commit. Can you ensure its working correctly?

**Comment by Jagadeesh-ajjada** [comment]
Yes it's working @itsnotmik i have tested.

**Comment by smrchanda877** [comment]
Yes, I tested the new changes, it's looking good. Do you think we should add input from the user for the question
In the last cell, we have put user_question as "I can't connect to wifi" as default.

Do you think it's a good idea to take user input there? instead of a default value

**Comment by itsnotmik** [comment]
I think a cell that can accept user input and query that response would be good, but for now a basic query is fine I believe. Its just to show in class

**Comment by ACraig7** [comment]
It seems to be working fine for me now. I used the command that has both ports in it. As @itsnotmik said, i must of pulled it at the wrong time or something else happened. Anything else need to be done or are we good to approve and merge?

**Comment by itsnotmik** [comment]
Id say for an initial work it is good and can be shown of in class. We can adjust and improve for next cycle

**Review by itsnotmik — APPROVED** [review]

**Review by itsnotmik — APPROVED** [review]

**Comment by itsnotmik** [comment]
@smrchanda877 please approve if it looks good for you

**Review by smrchanda877 — APPROVED** [review]
LGTM

**Pull request merged**